lib/brook/storage/redis.ex

```diff
@@ -118,6 +118,8 @@ defmodule Brook.Storage.Redis do
 
     put(instance, __MODULE__, %{namespace: namespace, redix: redix, event_limits: event_limits})
 
+    Brook.Storage.Redis.Migration.migrate(instance, redix, namespace)
```

Comment:
I'm trying to think through the implications of this running every time. If already migrated, I guess it's just a single call to redis and then empty map calls; I think that just makes sense...